From c880565f5c0f188ae060b1e219b12a60a538304f Mon Sep 17 00:00:00 2001
From: Cosimo Cecchi <cosimo@endlessm.com>
Date: Wed, 16 Oct 2013 15:30:19 -0700
Subject: [PATCH 3/3] Fix previous patchset

---
 src/eog-application.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/eog-application.c b/src/eog-application.c
index 737f978..37e75e6 100644
--- a/src/eog-application.c
+++ b/src/eog-application.c
@@ -31,6 +31,7 @@
 #include "eog-image.h"
 #include "eog-session.h"
 #include "eog-thumbnail.h"
+#include "eog-job-queue.h"
 #include "eog-window.h"
 #include "eog-application.h"
 #include "eog-application-activatable.h"
@@ -244,6 +245,7 @@ eog_application_startup (GApplication *application)
 	xmp_init();
 #endif
 	eog_debug_init ();
+	eog_job_queue_init ();
 	eog_thumbnail_init ();
 
 	/* Load special style properties for EogThumbView's scrollbar */
-- 
1.8.1.2

